### ISSUE TITLE
Two JAX-RS resources with the same name: one gets overwritten

### ISSUE THREAD

**Issue by stoicflame** [stoicflame/enunciate#280]
We have two resources that have the same name but are in different packages each identified with a version. Each have their own annotations.

For example:

```java
@ResourceLabel("AboutResource v1")
@Path("v1/about")
...v1.rest.admin.impl.AboutResourceIml.java
.
.
.

@ResourceLabel("AboutResource v2")
@Path("rest/v2/about")
...v2.rest.admin.impl.AboutResourceIml.java
.
.
.
```

After performing a mvn clean install I see both resources in the generated index.html but when I click on the the AboutResource v1 it takes me two the v2 documentation.  Specifically, it doesn't look like the AboutResource v1 details were ever generated.  There is only target/xxxx-SNAPSHOT/resource_AboutResourceImpl.html.  I would expect there would be two html files (one for each version of the resource).

**Comment by stoicflame** [comment]
Fixed at b94789f.

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
